lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,288 @@
+---
+name: Financial Transaction
```

Comment:
The body fields section is not getting displayed correctly. Need to probably revisit spacing in this file

Reply:
doesn't the body fields get rendered based on the financial_transaction file?

Reply:
ph thats the terminology binding

Reply:
It's from financial_transaction file. 
<img width="738" alt="Screenshot 2019-11-06 at 3 55 37 PM" src="https://user-images.githubusercontent.com/53420281/68290825-a0c81c80-00ae-11ea-8406-8d8cbdb7737f.png">

Is this how it should be displayed?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/de6e3d409009dba3a6db3d3b4479d449f9b9057c#diff-5125c86cde87f8439cb9d1bb859194e9R8-R23